portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,45 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="navbar">
+        <div>
+            <h1 class="name">L I L Y</h1>
```

Comment:
One thing I have to consider with my job is how my HTML will work with screen readers (for people who have low or no vision). I believe screen readers will ready out each individual letter here. If you *want* them to read out your name instead, you could use the css `letter-spacing` rule to achieve this effect: https://www.w3schools.com/cssref/pr_text_letter-spacing.asp

(To test this out, you can turn your phone's screen reader on in the accessibility settings!)

This is totally optional and up to you. It could be that you like the effect of it spelling your name, that's totally fine!

Reply:
Wow i had no idea! Thanks for letting me know:)